app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -0,0 +1,21 @@
+package com.intive.picover.common.di
+
+import com.google.firebase.storage.FirebaseStorage
+import dagger.Module
+import dagger.Provides
+import dagger.Reusable
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.Dispatchers
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AppModule {
+	@Provides
+	@Reusable
```

Comment:
"If you think a binding should be scoped for purely performance reasons, first verify that the performance is an issue, and if it is consider using @Reusable instead of a component scope."

Really performance is an issue here :D? I would suggest to use not use anything expect `@Provides` here, if the scoping/reusing will be needed we will add it later.

Reply:
done